packages/core/src/core/index.js

```diff
@@ -129,15 +132,20 @@ export const createExecuteCommand = ({
     const extractEdgeInfo = injectExtractEdgeInfo({ logger });
     const createResponse = injectCreateResponse({ extractEdgeInfo });
     const getLocationHint = injectGetLocationHint({ orgId, cookieJar });
+    const buildEndpointUrl = createBuildEndpointUrl({
+      getLocationHint,
+      getAssuranceValidationTokenParams,
+      apiVersion,
+      queryString,
```

Comment:
(nit) we could use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams) instead of `reactor-query-string`

Reply:
This is the util helper that is used across the library and in Tags, and I think it's good to have consistency.